draft-ietf-tls-external-psk-importer.md

```diff
@@ -178,34 +187,37 @@ IPSK with base key `ipskx` is computed as follows:
 
 L corresponds to the KDF output length of ImportedIdentity.target_kdf as defined in {{IANA}}.
 For hash-based KDFs, such as HKDF_SHA256(0x0001), this is the length of the hash function
-output, i.e., 32 octets. This is required for the IPSK to be of length suitable for supported
-ciphersuites.
+output, i.e., 32 octets for SHA256. This is required for the IPSK to be of length suitable
```

Comment:
Super nitpicky nitpick: should we s/i.e./e.g./? SHA-256 is still just an example.

Reply:
Yep!